src/generated/java/com/prowidesoftware/swift/model/mt/mt3xx/MT380.java

```diff
@@ -1223,6 +1247,18 @@ public static SequenceB1 getSequenceB1(SwiftTagListBlock parentSequence) {
 		}
 		return s;
 	}
+    /**
+	 * Get the single occurrence of SequenceB1 delimited by 16R/16S the value of SequenceB1#START_END_16RS.
```

Comment:
Notice this is not the only case to cover. You have the getSequenceList methods, and you have also the method variants without parent parameter. So there are four cases to cover and provide the alternative new API

Reply:
Ok